t/op/split.t

```diff
@@ -725,3 +725,37 @@ SKIP: {
 		}, "special-case pattern for $prog");
 	}
 }
+
+# gh18032: check that `split " "` does not get converted to `split ""`
+SKIP: {
+    skip_if_miniperl("special-case patterns: need dynamic loading", 5);
```

Comment:
The `5` should be `5*2 + 6*2`.

Reply:
thanks, fixed with dynamic computation along these lines.